docs/scripts/api-events.py

```diff
@@ -124,6 +124,7 @@ def category(name: str, desc: str, hooks: List[Type[hooks.Hook]]) -> None:
             tls.TlsClienthelloHook,
             tls.TlsStartClientHook,
             tls.TlsStartServerHook,
+            tls.TlsHandshakeHook,
```

Comment:
Any proposals how to make the naming somehow include the "completed" idea of this hook?
`TlsHandshakeCompletedHook` or similar? Or using the `...Start/End...` scheme?